planetarium/NineChronicles
# Update event banner

### Description

SSIA

### How to test

1. Check if event banners are displaying properly.
2. Check if banner dragging is blocked during animation.


### Required Reviewers
@planetarium/9c-dev 

### Screenshot
![image](https://user-images.githubusercontent.com/26648527/174527653-71f91c3b-f686-40c0-849e-2e27d1eb25d8.png)
